exercises/connect/examples/success-standard/src/Connect.hs

```diff
@@ -21,12 +21,12 @@ parseLines [] = error "Empty lines"
 parseLines ([]:_) = error "Empty first line"
 parseLines lines@(firstLine:_) =
   let height = length lines
-      width = length firstLine
+      width = length (filter (/= ' ') firstLine)
```

Comment:
in anticipation of the question "did you have to duplicate this", I guess not; I could have done `map (filter (/= ' ')) lines` in either `winner` or `parseLines`.

`winner` didn't seem quite right since the previous structure was that only `parseLines` cares about the particular string contents of the input, but I suppose I don't care enough to say I'd refuse to break that rule.

`parseLines` is doable, just `width` wouldn't be able to use `firstLine` anymore and would probably have to use `head lines'` or something.